server-code/Projects.js

```diff
@@ -27,6 +29,28 @@ function getAllProjects() {
     return buildTableHTML(data, "table-sm");
 }
 
+/**
+ * Returns HTML formatted list of all active projects.
+ * 
+ * @param {String} desiredProjectStatus – The string that represents the project status to watch for
+ * @return {String} – A constructed HTML table listing all active projects
+ */
+function getProjectTable(desiredProjectStatus) {
+    var data = getSheetInfo(MAIN_SHEET_ID_STR, PROJECTS_STR, DATA_STR);
+    var headers = data[0];
+    var projectStatusColIdx = findIdx("Project Status", headers);
+    headers = headers.slice(0, -1);
+    var activeProjects = [headers];
+    for (var rowIdx = 1; rowIdx < data.length; rowIdx++) {
+        projectStatus = data[rowIdx][projectStatusColIdx];
+        if (projectStatus == desiredProjectStatus) {
```

Comment:
Triple equals just to be safe
```suggestion
        if (projectStatus === desiredProjectStatus) {
```